src/js/translations/en-US.json

```diff
@@ -38,41 +38,58 @@
   "SERVICE_ACTIONS.SCALE_BY": "Scale By",
   "SERVICE_ACTIONS.SUSPEND": "Suspend",
 
-  "SERVICE_ACTIONS.SDK_GROUP_DELETE_1": "Copy and paste this command into the CLI",
+  "SERVICE_ACTIONS.SDK_GROUP_DELETE_1":
```

Comment:
Let's remove the things we no longer need.

Reply:
That is currently being used. I've removed the strings not used.